app/graphql/types/user_type.rb

```diff
@@ -43,5 +46,15 @@ def total_hours_mentored
     def mentees
       object.unique_mentees
     end
+
+    def my_rocks
+      user = User.find(object.id)
```

Comment:
You do not need to define a user variable since the user was already queried in the resolver in the query_type file. You just need to put object.pebbles or object.rocks (object is the user).

Reply:
Ohhhhh thanks for that!